plugin/storage/integration/elasticsearch_test.go

```diff
@@ -166,11 +172,14 @@ func testElasticsearchStorage(t *testing.T, allTagsAsFields bool) {
 			GetOperationsMissingSpanKind: true,
 		},
 	}
-	s.initializeES(t, allTagsAsFields)
+	s.initializeES(t, c, allTagsAsFields)
 	s.RunAll(t)
 }
 
 func TestElasticsearchStorage(t *testing.T) {
+	t.Cleanup(func() {
+		testutils.VerifyGoLeaksOnce(t)
```

Comment:
this will still fail because of goleaks, right? Let's exclude http transport goroutines just in this instance (not globally).

Reply:
I think then we have to skip package leak test for integration package for v1 or we can just skip for elastic search enviornment (that means individual checks for es and package checks for other backends)

Reply:
I am ok to repeat the same exclusion in the package check, because as long as we have individual test checks the package level check is just to silence the linter.